pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/thread/ThreadRuntime.java

```diff
@@ -149,7 +149,7 @@ private static ClassLoader loadJars(String jarFile,
             } catch (FileNotFoundException e) {
                 // this is usually like
                 // java.io.FileNotFoundException: /tmp/pulsar-nar/xxx.jar-unpacked/xxxxx/META-INF/MANIFEST.MF'
-                log.error("The file {} does not look like a .nar file", jarFile, e.toString());
```

Comment:
No need to dump the stack trace hereit was using toString intentionally
Otherwise the stacktracd will scare sys admins

Reply:
Ok, but we need a `{}`